src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,201 @@
+    operatingFrom: Optional[time] = Noneable(TimeValidator(time_format=TimeFormat.WITH_SECONDS))
+    operatingTo: Optional[time] = Noneable(TimeValidator(time_format=TimeFormat.WITH_SECONDS))
+    daysOfWeek: Optional[list[str]] = Noneable(ListValidator(EnumValidator(OpenDataSwissOperationTimeDaysOfWeek)))
+
+
+@validataclass
+class OpenDataSwissPropertiesInput:
+    operator: str = StringValidator()
+    displayName: str = StringValidator()
+    address: Optional[OpenDataSwissAddressInput] = Noneable(DataclassValidator(OpenDataSwissAddressInput))
+    capacities: list[OpenDataSwissCapacitiesInput] = ListValidator(DataclassValidator(OpenDataSwissCapacitiesInput))
+    additionalInformationForCustomers: Optional[OpenDataSwissAdditionalInformationInput] = Noneable(
+        DataclassValidator(OpenDataSwissAdditionalInformationInput)
+    )
+    parkingFacilityCategory: str = StringValidator()
+    parkingFacilityType: Optional[str] = Noneable(StringValidator())
+    salesChannels: Optional[list[str]] = Noneable(ListValidator(StringValidator()))
+    operationTime: Optional[OpenDataSwissOperationTimeInput] = Noneable(DataclassValidator(OpenDataSwissOperationTimeInput))
+
+    def __post_init__(self):
+        for capacity in self.capacities:
+            if capacity.categoryType == OpenDataSwissCapacityCategoryType.STANDARD:
+                return
+        # If no capacity with type PARKING was found, we miss the capacity and therefore throw a validation error
+        raise ValidationError(reason='Missing parking spaces capacity')
+
+    def get_osm_opening_hours(self) -> str:
+        check_counter_weekday: int = 0
+
+        # If any of opening_times From or To is null, conversion to OSM opening hours cannot be continued
+        if not self.operationTime.operatingFrom or not self.operationTime.operatingTo:
+            return None
+
+        # OSM 24/7 has no secs in its timeformat and no endtime 00:00, so we replace with 24:00 and remove the secs
+        opening_time: str = f'{self.operationTime.operatingFrom.strftime("%H:%M")}-{self.operationTime.operatingTo.strftime("%H:%M")}'
+        opening_time = '00:00-24:00' if opening_time == '00:00-00:00' else opening_time
+
+        for days_of_week_input in self.operationTime.daysOfWeek:
+            # If the weekday is within Monday to Friday, count to check later for OSM opening hours Mo-Fr
+            if days_of_week_input in list(OpenDataSwissOperationTimeDaysOfWeek)[:5]:
+                check_counter_weekday += 1
+
+        # If it's open all day and number of opening days is 7, then it is OSM - 24/7. No further handling needed in this case.
+        if opening_time == '00:00-24:00' and len(self.operationTime.daysOfWeek) == 7:
+            return '24/7'
+
+        osm_opening_hour: list = []
+        # If the days are Monday to Friday with same opening time, we can summarize it to the Mo-Fr entry,
+        # otherwise we have to set it separately
+        if check_counter_weekday == 5:
```

Comment:
Why not doing a direct check on the `daysOfWeek`? Like `all(day in self.operationTime.daysOfWeek for day in list(OpenDataSwissOperationTimeDaysOfWeek)[:5])` ? Then you don't need `check_counter_weekday` any more.